turbopack/crates/turbopack-browser/src/ecmascript/merged/update.rs

```diff
@@ -77,25 +77,25 @@ struct EcmascriptModuleEntry {
     #[serde(with = "turbo_tasks_fs::rope::ser_as_string")]
     code: Rope,
     url: String,
-    map: Option<String>,
+    #[serde(with = "turbo_tasks_fs::rope::ser_option_as_string")]
+    map: Option<Rope>,
 }
 
 impl EcmascriptModuleEntry {
     async fn from_code(id: &ModuleId, code: Vc<Code>, chunk_path: &str) -> Result<Self> {
         let map = match &*code.generate_source_map().await? {
             Some(map) => {
-                let map = map.await?.to_source_map().await?;
-                let mut map_str = vec![];
-                (*map).to_writer(&mut map_str)?;
```

Comment:
The old code generates SourceMaps that lead to warnings in Chrome since it emits `"url": null`